tests/Silex/Tests/ApplicationTest.php

```diff
@@ -12,6 +12,9 @@
 namespace Silex\Tests;
 
 use PHPUnit\Framework\TestCase;
+use ArrayObject;
+use Exception;
+use stdClass;
```

Comment:
I thought that we don't need to put `stdClass` in use section, but ok

Reply:
We do.